### ISSUE TITLE
No support for position -webkit-sticky as required by Safari

### ISSUE THREAD

**Issue by humphreybc** [typestyle/typestyle#198]
Apparently support for position “sticky” in Safari requires that you use `-webkit-sticky` instead as the value for the position property. [Looking at the Typestyle docs](https://typestyle.github.io/#/core/concept-fallbacks), I tried the following approach:

```
const className = style({
  position: ["-webkit-sticky", "sticky"],
  top: "0",
  zIndex: 999
});
```

However I get an error:

```
Types of property 'position' are incompatible.
  Type 'string[]' is not assignable to type '"initial" | "inherit" | "unset" | "revert" | "static" | "relative" | "absolute" | "sticky" | "fix...'.
    Type 'string[]' is not assignable to type '"fixed"'.
```

I also tried augmenting the types, but I think the docs are a bit out of date because I couldn't get that to work either. I'm quite new to Typescript.

**Comment by bradleyayers** [comment]
Added support in https://github.com/typestyle/typestyle/commit/70f585183830be1487de3478947251de55fc0ce1

@basarat is there any way for me to publish patch releases for changes like this?

**Comment by notoriousb1t** [comment]
Patched in v1.4.3
